src/screens/Requests/Requests.spec.tsx

```
@@ -248,33 +286,35 @@
+                <Requests />
+              </I18nextProvider>
+            </Provider>
+          </BrowserRouter>
+        </MockedProvider>,
+      );
+
+      await wait(200);
+      expect(container.textContent).not.toMatch(
+        'Organizations not found, please create an organization through dashboard',
+      );
+    });
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)